blocks/hero/product-hero.css

```diff
@@ -38,11 +38,10 @@
   left: 0;
   height: 100%;
   width: 100%;
-  object-position: center center;
+  object-position: center left;
```

Comment:
Because of the way the parent picture is defined, the value of this (left/center/right), it shouldn't have any impact on the position of the image.

Is this not the case?